packages/react-button/src/Playground.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { Checkbox, Dropdown, IDropdownOption, Stack, TextField } from '@fluentui/react';
```

Comment:
you won't be able to use here anything from v7/v8 , so this would need a bit more refactoring

Reply:
Agreed, we should probably change this for native select + input + style overrides.